app/components/plugin.tsx

```
@@ -313,18 +317,20 @@ export function PluginPage() {
             <List>
               <ListItem title={Locale.Plugin.EditModal.Content}>
-                <div className={pluginStyles["plugin-schema"]}>
+                <div className={pluginStyles['plugin-schema']}>
                   <input
                     type="text"
                     style={{ minWidth: 200 }}
-                    onInput={(e) => setLoadUrl(e.currentTarget.value)}
-                  ></input>
+                    onInput={e => setLoadUrl(e.currentTarget.value)}
+                  >
+                  </input>
```

Biome (1.9.4)

[error] input is a void element tag and must not have children.

Unsafe fix: Remove the children.


(lint/correctness/noVoidElementsWithChildren)